test/compflags/kushal/llvm-incremental.bad

```diff
@@ -0,0 +1 @@
+Incremental compilation is not yet supported with LLVM
```

Comment:
Because we aren't providing a line number, because this is a general error with the flag (which I should've remembered)